tex/chProgramming.tex

```diff
@@ -768,7 +776,7 @@ \subsection{Natural numbers}\index[concept]{natural number}
 
 When defining functions that operate on natural numbers, we can
 proceed by case analysis, as was done in the previous section for boolean
-values. Here as well, there are two cases: either the natural number used in
```

Comment:
I thought that  "here as well' was more idiomatic than "here also", but I trust your English better than mine :)